dev/io.openliberty.security.oidcclientcore.internal/resources/io/openliberty/security/oidcclientcore/internal/resources/OidcClientCoreMessages.nlsprops

```diff
@@ -111,3 +111,7 @@ JWK_CONNECTION_TIMED_OUT.useraction=Update the jwksConnectTimeout property in th
 JWK_READ_TIMED_OUT=CWWKS2422E: The OpenID Connect client failed to read data from the [{0}] JWK URI of the OpenID Connect provider within {1} milliseconds. Consider updating the jwksReadTimeout property in the OpenID Connect client configuration.
 JWK_READ_TIMED_OUT.explanation=The OpenID Connect client cannot read the JWK data from the JWK URI within the configured time frame.
 JWK_READ_TIMED_OUT.useraction=Update the jwksReadTimeout property in the OpenID Connect client configuration. Verify that the OpenID Connect provider is reachable.
+
+OIDC_CLIENT_INVALID_RESPONSE_TYPE=CWWKS2423E: Invalid response type ''{0}'' is specified. Valid response types are [{1}].
```

Comment:
```suggestion
OIDC_CLIENT_INVALID_RESPONSE_TYPE=CWWKS2423E: The following invalid response type is specified: {0}. Valid response types are [{1}].
```

Reply:
hi @dmuelle , i am wondering if it'd be okay to do:
`The following invalid response type is specified: ''{0}''. Valid response types are [{1}].`

i wanted to keep the quotes (they are single quotes, but need to do two single quotes in a row for it to display as one) in there because the response type can have spaces and i am worried that not having the quote would be confusing.

i.e., 
```
The following invalid response type is specified: 'id_token token'. Valid response types are ['code','code token'].
```
vs.
```
The following invalid response type is specified: id_token token. Valid response types are [code,code token].`

for the valid response types, i was planning on adding the quotes in the runtime, since it is a list.

note: the only valid one response type currently is `code`, but i added `code token` as well for the sake of the example.

Reply:
yes- if readability for the  response types is a concern, the quotes are ok. We generally try to avoid them but sometimes they are necessary